crates/pixi_manifest/src/task.rs

```diff
@@ -112,13 +110,10 @@ impl Task {
     }
 
     /// If this task is a plain task, returns the task string
-    pub fn as_plain(&self) -> Result<String, TaskStringError> {
+    pub fn as_plain(&self) -> Result<String, anyhow::Error> {
```

Comment:
I think you misunderstood one of my messages :)

Returning a custom error is the way to go, but the custom Error should be an enum instead of a struct